Gemfile

```diff
@@ -1,4 +1,9 @@
-source 'https://rubygems.org'
+source "https://rubygems.org"
 
 # Specify your gem's dependencies in sapience.gemspec
 gemspec
+
+gem "pry-nav"
+gem "sentry-raven"
+gem "statsd-ruby"
+gem "semantic_logger", github: "mhenrixon/semantic_logger"
```

Comment:
Just curious, why did you fork the `semantic_logger` gem, are you going to make some custom changes in it?